.travis.yml

```diff
@@ -28,6 +28,7 @@ deploy:
   skip_cleanup: true
   on:
     tags: true
+  tag: next
```

Comment:
Now releasing to `next` by default 💯

Reply:
so this means that every merge to master results in a release to `next`? If so, dope 👍

Reply:
No at the moment only when you push a tag it will create a release. But I suppose we could additionally push canary releases or something

Reply:
So then how do I release to 'next'? I have to delete the local, upstream tag 'next' tag, and then push a new 'next' tag?

Reply:
I think we are mising git and npm dist tags. For git tags nothing will change. We just publish to the npm dist tag next by default as a release channel. Releasing to the npm dist tag latest happens by moving a previously released version on the next channel to latest. Not through git but through the npm CLI.

Reply:
Yes, like Tobi said. To summarize it again:

- you create a git tag manually
- push the tag
- travis will then trigger the release to npm using the tag version and the dist-tag `next`
- once you're happy with the release (e.g. you tested it in one of the apps), you manually "move" the dist-tag to `latest` using the `npm dist-tag add @commercetools-frontend/ui-kit@<version> latest` command

Reply:
The good thing is that we can use from next and specify versions in our pkg.json normally. While nobody (incl us) will get notified through e.g. a bot about the new version being on next cause latest is the default channel. Unless the also pull specific versions or specify @latest in their pkg.json. It also means that versions will likely jump on the latest channel given we do not move them often. That however should not be a big deal.

Reply:
Hmm... I still don't understand.

Can you give the steps that we as a developer would go through (omitting everything done on by / on CI)?

Reply:
Assume we have v1.9.0 on latest on npm
Given you tag with git as v2.0.0 
Then v2.0.0 will be publish on CI to the next dist tag
Given you then merge some PRs
And then tag with git to v2.1.0
Then CI will publish v2.1.0 to next
Given now v2.1.0 should be the official release on latest on npm
Then you use npm‘s CLI and the dist-tag command to move the v2.1.0 from next to latest
Then latest on npm moved from v1.9.0 to v2.1.0

So the only manual steps is using git tags to have CI publish to next. While moving a npm tag from next to latest is like making it „official“. 

It‘s important to not mix git tags being used to auto publish to npms latest tag. While moving npm tags from latest to next has nothing todo with git tags again. 

Hope that helped.